nservicebus/event-streaming/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Event stream processing
+summary: Combining the Particular Service Platform with an event stream processor (Apache Kafka, Azure Event Hubs, or Amazon Kinesis)
+callsToAction:
+  - interest-apache-kafka
+  - interest-azure-event-hubs
+  - interest-amazon-kinesis
+reviewed: 2024-02-06
+related:
+  - samples/azure-functions/service-bus-kafka
+---
+
+_Event stream processing_ is a practice of taking action on a series of datapoints that originate from a system that continuously creates data.
+
+There are great tools that facilitate event streaming including:
+
+- [Apache Kafka](https://kafka.apache.org/)
+- [Azure Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/)
+- [Amazon Kinesis](https://aws.amazon.com/kinesis/).
+
+Although these tools and the Particular Service Platform both use the term _event_, [an event stream is not a queue](https://particular.net/blog/lets-talk-about-kafka).
```

Comment:
I am unsure if we should have a link to somewhere else so early. I guess I am concerned the user may click on that and move on.
Could links like this be added as "Other resources" at the end of the page?

Reply:
I was also concerned by this. Also about biasing the type of feedback people might give. If we state an strong opinion we preclude someone who thinks they really want a Kafka transport.